classes/Loggly.cls

```diff
@@ -462,32 +462,50 @@ global with sharing class Loggly {
 	*
 	* @param logs A set of log messages (in JSON format) to send to endpoint
 	*/
-	@future (callout=true)
 	global static void sendLogs(Set<String> logs) {
-		List<String> logList = new List<String>();
-		logList.addAll(logs);
+		try {
+			if (Limits.getQueueableJobs() < Limits.getLimitQueueableJobs()) {
+				System.enqueueJob(new QueueableLogSender(logs));
+			} else {
+				System.debug('Reached queueable limit');
+			}
+		} catch (Exception e) {
+			System.debug(e);
+		}
+	}
 
-		HttpResponse res = new HttpResponse();
-		// Build up a newline-separated list of JSON messages to send to the endpoint
-		String message = String.join(logList, '\n');
+	private class QueueableLogSender implements Queueable, Database.AllowsCallouts {
+		private Set<String> logs;
 
-		try {
-			HttpRequest req = new HttpRequest();
+		public QueueableLogSender(Set<String> logs) {
+			this.logs = logs;
+		}
 
-			if (THROW_TEST_EXCEPTION) {
-				throw new TestException('Test exception');
-			}
+		public void execute(QueueableContext context) {
+			List<String> logList = new List<String>(logs);
 
-			req.setEndpoint(LOGGLY_ENDPOINT);
-			req.setMethod('POST');
-			req.setHeader('content-type', 'application/json');
-			req.setBody(message);
+			HttpResponse res = new HttpResponse();
+			// Build up a newline-separated list of JSON messages to send to the endpoint
+			String message = String.join(logList, '\n');
 
-			Http http = new Http();
-			res = http.send(req);
-		} catch (Exception e) {
-			System.debug(e);
-			System.debug(res.toString());
+			try {
+				HttpRequest req = new HttpRequest();
+
+				if (THROW_TEST_EXCEPTION) {
+					throw new TestException('Test exception');
+				}
+
+				req.setEndpoint(LOGGLY_ENDPOINT);
+				req.setMethod('POST');
+				req.setHeader('content-type', 'application/json');
+				req.setBody(message);
+
+				Http http = new Http();
+				res = http.send(req);
+			} catch (Exception e) {
+				System.debug(e);
```

Comment:
I know that you didn't write this bit, but can you update it to have a better logging message?  Old me was dumb.

```java
List<String> errorMessages = new List<String> {
    'SLoggly: Error sending message to Loggly Endpoint "' + LOGGLY_ENDPOINT + '"',
    e.getTypeName() + ': ' + e.getMessage(),
    e.getStackTraceString()
};

System.debug(System.LoggingLevel.ERROR, String.join(errorMessages, '\n'));
```